al-khaser/Anti Debug/ParentProcess.cpp

```diff
@@ -50,6 +50,7 @@ DWORD GetParentProcessId()
 
 BOOL IsParentExplorerExe()
 {
+	//NOTE this check is wank because you can use an alternative file manager

```

Comment:
true, thanks for pointing this, in addition to that, malware runs through several stages, so parent processes would not be always explorer.exe.